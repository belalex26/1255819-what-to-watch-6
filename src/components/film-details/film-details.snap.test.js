import React from 'react';
import {Router} from 'react-router-dom';
import {createMemoryHistory} from 'history';
import {render} from '@testing-library/react';

import FilmDetails from './film-details';
import {mockFilm} from '../../test-mocks';

it(`FilmDetails renders correctly`, () => {
  const history = createMemoryHistory();
  const {container} = render(
      <Router history={history}>
        <FilmDetails
          film={mockFilm}
        />
      </Router>
  );
  expect(container).toMatchSnapshot();
});
