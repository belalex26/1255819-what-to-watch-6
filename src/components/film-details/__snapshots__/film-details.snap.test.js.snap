// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilmDetails renders correctly 1`] = `
<div>
  <div
    class="movie-card__text movie-card__row"
  >
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Director
        </strong>
        <span
          class="movie-card__details-value"
        >
          William Friedkin
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Starring
        </strong>
        <span
          class="movie-card__details-value"
          style="white-space: pre-line;"
        >
          Linda Blair, 
Ellen Burstyn, 
Jason Miller, 
Max von Sydow
        </span>
      </p>
    </div>
    <div
      class="movie-card__text-col"
    >
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Run Time
        </strong>
        <span
          class="movie-card__details-value"
        >
          2h 8m
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Genre
        </strong>
        <span
          class="movie-card__details-value"
        >
          Thriller
        </span>
      </p>
      <p
        class="movie-card__details-item"
      >
        <strong
          class="movie-card__details-name"
        >
          Released
        </strong>
        <span
          class="movie-card__details-value"
        >
          1973
        </span>
      </p>
    </div>
  </div>
</div>
`;
